### DISCUSSION TITLE
How to use "helm install" and not "helm upgrade --install"

### DISCUSSION THREAD

**Discussion by rabbagliettiandrea** [helmfile/helmfile#875] [Questions]
Hello there,
a chart that I'm using shows different behavior if you use "helm install" instead of "helm upgrade --install".

Is there a way to issue "helm install" with helmfile?

Thank you so much,
Andrea

**Comment by yxxhero** [comment]
@rabbagliettiandrea "helm install" is for the first time chart install.  could you show the different behavior so that I can try to solve your issue.

**Comment by rabbagliettiandrea** [comment]
Hi @yxxhero, thank you for your response :)

If you could look here:
https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.5/deploy/installation/

and specifically here:
![image](https://github.com/helmfile/helmfile/assets/579861/a7dad0af-04e7-4434-9c10-cb13b29261e3)

it states that if you do "helm upgrade" the chart will not install any CRDs, and in fact it raises error on "helmfile apply" that's because it issue "helm upgrade --install"

**Comment by jouve** [comment]
` helm upgrade --install`  runs install internally the [first time](https://github.com/helm/helm/blob/main/cmd/helm/upgrade.go#L110). 

the "tip" is not related to helm install/upgrade, but how helm handle CRD that are defined in the charts ` crds/`  folder:

[ref](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#some-caveats-and-explanations)

>  There is no support at this time for upgrading or deleting CRDs using Helm.
